### PR TITLE
(Update) HEC Connector information

### PR DESCRIPTION
**Pull request by KinnonYee-harness**
## Description

* Please describe your changes: \___Updating HEC Connector information, along with screenshot, to help with some troubleshooting steps/issues
* Jira/GitHub Issue numbers (if any): \____[__________________________](https://harnesssupport.zendesk.com/agent/tickets/64534)
* Preview links/images (Internal contributors only): \___Added to Static folder

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 1]
```diff
@@ -116,6 +116,15 @@ To configure the Splunk streaming connector:
 
    The streaming destination gets configured and appears in the list of destinations under **Audit Log Streaming**. By default the destination is inactive.
 
+:::HEC Configuration Troubleshooting
```
Comment (brian-f-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:119]:
```suggestion
::: info troubleshoot HEC configurations
```

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 2]
```diff
@@ -116,6 +116,15 @@ To configure the Splunk streaming connector:
...
+Customers may encounter some issues with the HEC connector.  Here are some common troubleshooting steps.
```
Comment (brian-f-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:120]:
```suggestion
At times, you might issues with the HEC connector. Here are some troubleshooting steps you can take to resolve common issues.
```

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 3]
```diff
@@ -116,6 +116,15 @@ To configure the Splunk streaming connector:
...
+1. HEC connector should be set without a declared `sourcetype`.  Although the data will be JSON-Formatted, it is not declared as such.  Setting a sourcetype as `JSON` may filter data from the stream.
```
Comment (brian-f-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:121]:
```suggestion
1. You should set HEC connectors without a declared `sourcetype`. Although the data will be JSON-formatted, it is not declared as such. Setting a `sourcetype` as `JSON` might filter data from the stream.
```

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 4]
```diff
@@ -116,6 +116,15 @@ To configure the Splunk streaming connector:
...
+![](../../governance/audit-trail/static/hec_connector_notype.png)
+2. Harness utilzies the [following standard endpoints](https://docs.splunk.com/Documentation/Splunk/9.2.1/Data/HECRESTendpoints), and are automatically appended to the customer's URL.  They are not customizable.
```
Comment (brian-f-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:123]:
```suggestion
2. Harness utilizes the [following standard endpoints](https://docs.splunk.com/Documentation/Splunk/9.2.1/Data/HECRESTendpoints) and appends them automatically to the customer's URL. They are not customizable.
```

**Review by brian-r-fisher — DISMISSED** [review]

**Comment by brian-f-harness** [comment]
@piyush-harness could you please review/approve? Thanks.

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 5]
```diff
@@ -116,6 +116,15 @@ To configure the Splunk streaming connector:
...
 
+::: info troubleshoot HEC configurations
+At times, you might issues with the HEC connector. Here are some troubleshooting steps you can take to resolve common issues.
+1. You should set HEC connectors without a declared `sourcetype`. Although the data will be JSON-formatted, it is not declared as such. Setting a `sourcetype` as `JSON` might filter data from the stream.
+![](../../governance/audit-trail/static/hec_connector_notype.png)
+2. Harness utilizes the [following standard endpoints](https://docs.splunk.com/Documentation/Splunk/9.2.1/Data/HECRESTendpoints) and appends them automatically to the customer's URL. They are not customizable.
+/services/collector/event
+/services/collector/health
+/services/server/info
+:::
```
Comment (brian-f-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:127]:
```suggestion
::: info troubleshoot HEC configurations
At times, you might issues with the HEC connector. Here are some troubleshooting steps you can take to resolve common issues.

- You should set HEC connectors without a declared `sourcetype`. Although the data will be JSON-formatted, it is not declared as such. Setting a `sourcetype` as `JSON` might filter data from the stream.

   ![](../../governance/audit-trail/static/hec_connector_notype.png)

- Harness utilizes the [following standard endpoints}(https://docs.splunk.com/Documentation/Splunk/9.2.1/Data/HECRESTendpoints) and appends them automatically to the customer's URL. They are not customizable.
   - `/services/collector/event`
   - `/services/collector/health`
   - `/services/server/info`
:::
```

**Review by brian-r-fisher — DISMISSED** [review]

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 6]
```diff
@@ -116,6 +116,18 @@ To configure the Splunk streaming connector:
 
    The streaming destination gets configured and appears in the list of destinations under **Audit Log Streaming**. By default the destination is inactive.
 
+::: info troubleshoot HEC configurations
+At times, you might issues with the HEC connector. Here are some troubleshooting steps you can take to resolve common issues.
+
+- You should set HEC connectors without a declared `sourcetype`. Although the data will be JSON-formatted, it is not declared as such. Setting a `sourcetype` as `JSON` might filter data from the stream.
```
Comment (piyush-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:122]: I think this is more of a current implementation limitation, We can have an enhancement  for customer to configure SourceType on Splunk connector and send it as part of Request.

Comment (brian-f-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:122]:
```suggestion
- You should set HEC connectors without a declared `SourceType`. Although the data will be JSON-formatted, it is not declared as such. Setting a `SourceType` as `JSON` might filter data from the stream.
```

Comment (KinnonYee-harness) [reply docs/platform/governance/audit-trail/audit-streaming.md]: Hi Piyush.
I'm good with that, but currently, this is the needed configuration.  We can update this documentation again after we complete the enhancement

Comment (KinnonYee-harness) [reply docs/platform/governance/audit-trail/audit-streaming.md]: Created Jira to track Enhancement request:
https://harness.atlassian.net/browse/PL-52052

**Review by brian-r-fisher — DISMISSED** [review]

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 7]
```diff
@@ -116,6 +116,18 @@ To configure the Splunk streaming connector:
 
    The streaming destination gets configured and appears in the list of destinations under **Audit Log Streaming**. By default the destination is inactive.
 
+::: info troubleshoot HEC configurations
+At times, you might issues with the HEC connector. Here are some troubleshooting steps you can take to resolve common issues.
```
Comment (brian-f-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:120]:
```suggestion
At times, you might experience issues with the HEC connector. Here are some troubleshooting steps you can take to resolve common issues.
```

**Review by brian-r-fisher — DISMISSED** [review]

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 8]
```diff
@@ -116,6 +116,18 @@ To configure the Splunk streaming connector:
 
    The streaming destination gets configured and appears in the list of destinations under **Audit Log Streaming**. By default the destination is inactive.
 
+::: info troubleshoot HEC configurations
```
Comment (brian-f-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:119]:
```suggestion
:::info troubleshoot HEC configurations
```

File: docs/platform/governance/audit-trail/audit-streaming.md [hunk 9]
```diff
@@ -116,6 +116,18 @@ To configure the Splunk streaming connector:
...
    The streaming destination gets configured and appears in the list of destinations under **Audit Log Streaming**. By default the destination is inactive.
 
+:::info troubleshoot HEC configurations
+At times, you might experience issues with the HEC connector. Here are some troubleshooting steps you can take to resolve common issues.
+
+- You should set HEC connectors without a declared `SourceType`. Although the data will be JSON-formatted, it is not declared as such. Setting a `SourceType` as `JSON` might filter data from the stream.
+
+   ![](../../governance/audit-trail/static/hec_connector_notype.png)
+
+- Harness utilizes the [following standard endpoints}(https://docs.splunk.com/Documentation/Splunk/9.2.1/Data/HECRESTendpoints) and appends them automatically to the customer's URL. They are not customizable.
```
Comment (brian-f-harness) [inline docs/platform/governance/audit-trail/audit-streaming.md:126]:
```suggestion
- Harness utilizes the following [standard endpoints](https://docs.splunk.com/Documentation/Splunk/9.2.1/Data/HECRESTendpoints) and appends them automatically to the customer's URL. They are not customizable.
```

**Review by brian-r-fisher — APPROVED** [review]

**Pull request merged**